File: Java Programs/lib/LinkedListArray.java
package lib;

public class LinkedListArray {
    int data;
    LinkedListArray next;

    public  LinkedListArray(){
        data = -999;
        next = null;
    }
    public int getData() {
        return data;
    }
    public void setData(int data) {
        this.data = data;
    }
    public LinkedListArray getNext() {
        return next;
    }
    public void setNext(LinkedListArray next) {
        this.next = next;
    }
}

File: PythonProjects/DataStructures/lib/linkedListArray.py
class LinkedListArray():
    def __init__(self):
        self.data = -999
        self.next_addr = None

    def get_data(self):
        return self.data

    def set_data(self, data):
        self.data = data

    def get_next_addr(self):
        return self.next_addr

    def set_next_addr(self, next_addr):
        self.next_addr = next_addr
